app/controllers/halls_controller.rb

```diff
@@ -6,7 +6,12 @@ def index
     render json: Halls::Representer.new(halls).basic
   end
 
+  def new
```

Comment:
This should be action `create`, not `new`. Please read about difference [here](https://stackoverflow.com/a/2472489)

Reply:
Okey looking into the diffrene